PopPUNK/plot.py

```diff
@@ -477,8 +477,11 @@ def writeClusterCsv(outfile, nodeNames, nodeLabels, clustering, output_format =
     # process clustering data
     nodeLabels = [r.split('/')[-1].split('.')[0] for r in nodeNames]
 
+    # get example clustering name for validation
+    example_cluster_title = list(clustering.keys())[0]
```

Comment:
This is a good change, but can tidy this up a little as we don't really use this variable